src/app/pages/OpenWalletPage/Features/ImportAccountsSelectionModal/index.tsx

```diff
@@ -44,32 +49,71 @@ function ImportAccountsSelector({ accounts }: ImportAccountsSelectorSelectorProp
 
 interface ImportAccountsSelectionModalProps {
   abort: () => void
-  type: 'mnemonic' | 'ledger'
+  type: WalletType.Mnemonic | WalletType.Ledger
 }
 
 export function ImportAccountsSelectionModal(props: ImportAccountsSelectionModalProps) {
   const { t } = useTranslation()
+  const size = useContext(ResponsiveContext)
   const importAccounts = useSelector(selectImportAccounts)
+  const accounts = useSelector(selectImportAccountsOnCurrentPage)
+  const pageNum = useSelector(selectImportAccountsPageNumber)
   const error = importAccounts.error
   const selectedAccounts = useSelector(selectSelectedAccounts)
   const dispatch = useDispatch()
 
   const openAccounts = () => {
     dispatch(
-      props.type === 'ledger'
+      props.type === WalletType.Ledger
         ? walletActions.openWalletsFromLedger()
         : walletActions.openWalletFromMnemonic(),
     )
   }
-  const cancelDisabled = importAccounts.step === ImportAccountsStep.Done || error ? false : true
-  const confirmDisabled = importAccounts.step !== ImportAccountsStep.Done || selectedAccounts.length === 0
+
+  const isBusyImporting = importAccounts.step !== ImportAccountsStep.Idle
+  const cancelDisabled = isBusyImporting && !error
+  const confirmDisabled = isBusyImporting || selectedAccounts.length === 0
+
+  const canGoPrev = !isBusyImporting && !!pageNum && !error
+
+  const onPrev = () => {
+    dispatch(importAccountsActions.setPage(pageNum - 1))
+  }
+
+  const canGoNext = !isBusyImporting && pageNum !== numberOfAccountPages - 1 && !error
+
+  const onNext = () => {
+    dispatch(importAccountsActions.setPage(pageNum + 1))
+    if (props.type === 'ledger') {
+      dispatch(importAccountsActions.enumerateMoreAccountsFromLedger())
+    }
+  }
+
+  let selectionLabel: string
+  switch (selectedAccounts.length) {
+    case 0:
+      selectionLabel = t('openWallet.importAccounts.noAccountSelected', 'No account selected')
+      break
+    case 1:
+      selectionLabel = t('openWallet.importAccounts.oneAccountSelected', 'One account selected')
+      break
+    default:
+      selectionLabel = t(
+        'openWallet.importAccounts.multipleAccountsSelected',
+        '{{ number }} accounts selected',
+        { number: selectedAccounts.length },
+      )
+  }
```

Comment:
This looks like reimplementation of plurals https://www.i18next.com/translation-function/plurals

Reply:
We don't have any existing plurals, and i18next-scanner doesn't seem to support it correctly via
```tsx
t('aa', {
  count: count,
  defaultValue_0: 'zero1',
  defaultValue_zero: 'zero2',
  defaultValue_1: 'one1',
  defaultValue_one: 'one2',
  defaultValue: 'plural1 {{count}}',
  defaultValue_plural: 'plural2 {{count}}',
  defaultValue_other: 'plural3 {{count}}',
})
```

I came up with this workaround:
```tsx
  // Workaround for i18next-scanner to pickup plurals correctly, because it is missing
  // defaultValue_zero, defaultValue_one, defaultValue_other here:
  // https://github.com/i18next/i18next-scanner/blob/4687b6a/src/parser.js#L502-L503
  t('aa_zero', 'zero')
  t('aa_one', 'one')

  return <div>{t('aa', 'count {{count}}', { count: count })}</div>
```

Reply:
OK, I have implemented it using this workaround.